package.json

```diff
@@ -27,6 +27,7 @@
     "commitizen": "^4.2.5",
     "cz-conventional-changelog": "^3.2.0",
     "depcheck": "1.4.3",
+    "esbuild": "0.15.10",
```

Comment:
esbuild is the current tool suggested by the vscode docs for bundling the js file.  Works like a champ so far.